star-dash/star-dash/Constants/SpriteConstants.swift

```diff
@@ -20,6 +20,7 @@ struct TextureSet {
 
 struct SpriteConstants {
     static let PlayerRedNose = "PlayerRedNose"
+    static let PlayerRedNoseLeft = "PlayerRedNose"
```

Comment:
What was this introduced for?

Reply:
I iniitally wanted to the player to stay looking left after moving left, but didn't know how to and gave up.

Its irrelevant now. deleted.